src/containers/EmptyGraphLayoutContainer.tsx

```diff
@@ -75,17 +86,34 @@ export class EmptyGraphLayout extends React.Component<EmptyGraphLayoutProps, Emp
     const isGraphEmpty = !this.props.elements || !this.props.elements.nodes || this.props.elements.nodes.length < 1;
 
     if (isGraphEmpty) {
+      console.log(this.props.isDisplayingUnusedNodes);
       return (
         <EmptyState className={emptyStateStyle}>
           <EmptyStateTitle>Empty Graph</EmptyStateTitle>
           <EmptyStateInfo>
             There is currently no graph available for namespace <b>{this.props.namespace}</b>. This could either mean
-            there are no service mesh available in this namespace or that nothing has accessed the service mesh. Please
-            try accessing something in the service mesh and click 'Refresh'.
+            there are no service mesh available in this namespace or that nothing has accessed the service mesh.
```

Comment:
Requested change: "...there is no service mesh available for this namespace..."

Suggested: maybe also change "accessed" to "request traffic": "...there is no service mesh available for this namespace or the service mesh has yet to see request traffic."